src/popup/pages/PermissionsSettings.vue

```diff
@@ -34,6 +35,9 @@
 </template>
 
 <script>
```

Comment:
Suggest to add `lang="ts"` in `script`.

Reply:
I believe this is out of the scope of this migration and can be done in a separate PR